src/index.js

```diff
@@ -1,2 +1,36 @@
-import './style.css'
+import './style.css';
+import logo from './assets/images/logo.png';
+import { getPokemonList } from './displaypokemonitems.js';
+import { getLike } from './popup.js';
 
+const myLogo = document.getElementById('myLogo');
+const likeAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/oheYik9wd8sLSwZdAJ1P/likes';
+// Add Logo Image //
+let imageHtml = '';
+imageHtml += `
+<img src="${logo}" alt="logo">`;
```

Comment:
- [ ] To follow the YAGNI principle, please kindly delete extra image files in your code. I notice there are 3 logo.png files and unused pokemon images. Please kindly remove unused images.